tests/js/spec/components/dropdownReact.spec.jsx

```diff
@@ -78,11 +78,6 @@ describe('DropdownReact', function() {
         expect(wrapper.state('isOpen')).toBe(false);
       });
 
-      it('closes when dropdown menu item is clicked', function() {
```

Comment:
Why are we removing this test case, seems like a reasonable test?

Reply:
We don't necessarily want the dropdown to close when any item in the menu is clicked. In a nested dropdown we don't want a click on a top level item to do anything, only on the second level which has some action associated with it.

Reply:
Right, but shouldn't we support the case where we don't have nested menus and we want the dropdown to close when an item is clicked?

Reply:
Makes sense. I'm gonna drop this change from the PR then